test/client/fixtures/automation/click-test.js

```diff
@@ -715,6 +715,43 @@ $(document).ready(function () {
             });
     });
 
+    if (!browserUtils.isIE) {
+        asyncTest('click and mouseup events should have equal `timeStamp` properties', function () {
+            const target = document.createElement('div');
+
+            target.className    = TEST_ELEMENT_CLASS;
+            target.style.width  = '10px';
+            target.style.height = '10px';
+
+            document.body.appendChild(target);
+
+            let mouseUpTimeStamp = null;
+            let clickTimeStamp   = null;
+
+            target.addEventListener('mouseup', function (e) {
+                mouseUpTimeStamp = e.timeStamp;
+            });
+
+            target.addEventListener('click', function (e) {
+                clickTimeStamp = e.timeStamp;
+            });
+
+            const clickAutomation = new ClickAutomation(target, { });
+
+            return clickAutomation
+                .run()
+                .then(function () {
+                    ok(!!mouseUpTimeStamp);
```

Comment:
You check that `mouseUpTimeStamp` is a valid timestamp and not equal `null`.
In my opinion, it's more readable to use this notation:
```js
ok (typeof `mouseUpTimeStamp` === 'number');
```